src/net.cpp

```diff
@@ -1671,18 +1671,18 @@ void CConnman::ThreadSocketHandler()
 
     while (!interruptNet)
     {
-        if (GetTime() - nLastDisconnectNodes > 1) {
+        if (GetTimeMillis() - nLastDisconnectNodes > 1000) {
```

Comment:
nit: Should this be 1 * 1000? Does it matter for consistency sake? 
```suggestion
        if (GetTimeMillis() - nLastDisconnectNodes > 1 * 1000) {
```

Reply:
Second thinking this... Noone is going to get the impression it could be 1000 seconds. Just ignore me 😂